lib/gc/ExecutionEngine/CPURuntime/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+find_package(OpenMP REQUIRED)
+
+if ("iomp" IN_LIST OpenMP_C_LIB_NAMES OR "omp" IN_LIST OpenMP_C_LIB_NAMES OR "omp5" IN_LIST OpenMP_C_LIB_NAMES)
```

Comment:
Does this match "libiomp", "libiomp5", "libomp", "libomp5"?

Reply:
Yes it should. :)

Reply:
> Yes it should. :)

If so, then the `"omp" IN_LIST OpenMP_C_LIB_NAMES` will also match `libgomp`? And this is not what we want?

Reply:
> then the "omp" IN_LIST OpenMP_C_LIB_NAMES will also match libgomp

No, I suppose. libgomp has the name `gomp` in `OpenMP_C_LIB_NAMES`.  `IN_LIST` of cmake performs a full-name search, instead of finding substr.

> "libiomp", "libiomp5", "libomp", "libomp5"

It should only match `iomp omp omp5`